src/app.js

```diff
@@ -11,6 +12,7 @@ const port = process.env.PORT || 3000;
 app.use(bodyParser.urlencoded({ extended: true }));
 
 app.post("/doesheknower", barelyBot);
+app.post("/doesheknower", flippyBot);
```

Comment:
good call, but isn't that going to require another slack integration? We are already at our limit :(